src/system/data-attributes-customer.md

```diff
@@ -6,7 +6,7 @@ The following tables list the attributes from a typical export of the Customers
 
 Each attribute, or field, is represented in the CSV file as a column, and customer records are represented by rows. Columns that begin with an underscore are service entities that contain properties or complex data.
 
-### Customers Main File
+## Customers Main File
```

Comment:
some of these fields are defined in devdocs.  the ones i found are below if you want to add them.

email - The customer's email address.
dob - The customer’s date of birth.
firstname - The customer’s first name.
gender - The customer’s gender.
group_id - The group assigned to the user. Default values are 0 (Not logged in), 1 (General), 2 (Wholesale), and 3 (Retailer).
lastname - The customer’s family name.
middlename - The customer’s middle name.
prefix - An honorific, such as Dr., Mr., or Mrs.
suffix - A value such as Sr., Jr., or III.
taxvat - The customer’s Tax/VAT number (for corporate customers).

Customer Addresses
city - The customer's city or town.
company - The customer's company.
country_id - The customer's country.
fax - The customer's fax number.
firstname - The first name of the person associated with the shipping/billing address.
lastname - The family name of the person associated with the shipping/billing address.
middlename - The middle name of the person associated with the shipping/billing address.
postcode - The customer’s ZIP or postal code.
prefix - An honorific, such as Dr., Mr., or Mrs.
region - An object that defines the customer’s state or province.
region_id - A number that uniquely identifies the state, province, or other area.
street - The number and name of the street associated with the shipping/billing address.
suffix - A value such as Sr., Jr., or III.
telephone - The customer's telephone number.
vat_id - The customer’s Tax/VAT number (for corporate customers).